### DISCUSSION TITLE
最新的Ubuntu无法打开城市天际线和造(tan)桥(wu)模拟器

### DISCUSSION THREAD

**Discussion by Nofated095** [Gamuxorg/bbs#324] [游戏或网站运行反馈]
游戏名称：

操作系统：*Ubuntu 20.10 x64*

bug描述：*打不开citiesskylines_1.7.0-f5_amd64和PolyBridge_amd64*

终端运行输出 ：
```
Set current directory to /tmp/.mount_VfD2Xh/usr/bin
Found path: /tmp/.mount_VfD2Xh/usr/bin/Cities.x64
Mono path[0] = '/tmp/.mount_VfD2Xh/usr/bin/Cities_Data/Managed'
Mono path[1] = '/tmp/.mount_VfD2Xh/usr/bin/Cities_Data/Mono'
Mono config path = '/tmp/.mount_VfD2Xh/usr/bin/Cities_Data/Mono/etc'
displaymanager : xrandr version warning. 1.6
client has 2 screens
displaymanager screen (0)(LVDS-1): 1280 x 800
Using libudev for joystick management


Importing game controller configs
```
```
Set current directory to /tmp/.mount_PolyBrgRHhtX/usr/bin
Found path: /tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge.x86_64
Mono path[0] = '/tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge_Data/Managed'
Mono path[1] = '/tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge_Data/Mono'
Mono config path = '/tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge_Data/Mono/etc'
displaymanager : xrandr version warning. 1.6
client has 2 screens
displaymanager screen (0)(LVDS-1): 1280 x 800
Using libudev for joystick management


Importing game controller configs
```

自行尝试解决bug的方式和结果：*已经尝试通过[公告](https://github.com/Gamuxorg/bbs/issues/97)中所给出的方案全部进行了安装*

**Comment by hyh19962008** [comment]
好像是多显示器的问题？

Best

________________________________

**Comment by Nofated095** [comment]
> 好像是多显示器的问题？ Best
> […](#)
> ________________________________ From: Nofated <notifications@github.com> Sent: Wednesday, February 24, 2021 4:31:17 PM To: Gamuxorg/bbs <bbs@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: [Gamuxorg/bbs] 最新的Ubuntu无法打开城市天际线和造(tan)桥(wu)模拟器 (#141) 游戏名称： 操作系统：Ubuntu 20.10 x64 bug描述：打不开citiesskylines_1.7.0-f5_amd64和PolyBridge_amd64 终端运行输出 ： Set current directory to /tmp/.mount_VfD2Xh/usr/bin Found path: /tmp/.mount_VfD2Xh/usr/bin/Cities.x64 Mono path[0] = '/tmp/.mount_VfD2Xh/usr/bin/Cities_Data/Managed' Mono path[1] = '/tmp/.mount_VfD2Xh/usr/bin/Cities_Data/Mono' Mono config path = '/tmp/.mount_VfD2Xh/usr/bin/Cities_Data/Mono/etc' displaymanager : xrandr version warning. 1.6 client has 2 screens displaymanager screen (0)(LVDS-1): 1280 x 800 Using libudev for joystick management Importing game controller configs Set current directory to /tmp/.mount_PolyBrgRHhtX/usr/bin Found path: /tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge.x86_64 Mono path[0] = '/tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge_Data/Managed' Mono path[1] = '/tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge_Data/Mono' Mono config path = '/tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge_Data/Mono/etc' displaymanager : xrandr version warning. 1.6 client has 2 screens displaymanager screen (0)(LVDS-1): 1280 x 800 Using libudev for joystick management Importing game controller configs 自行尝试解决bug的方式和结果：已经尝试通过公告<#97>中所给出的方案全部进行了安装 ― You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub<#141>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFRK4UOQUCAL4RDMBRRLNPDTAS2NLANCNFSM4YEB5J6A>.

似乎并不是，因为我可以运行的其他游戏终端开始的输出和这个完全一样

**Comment by Nofated095** [comment]
> 好像是多显示器的问题？ Best
> […](#)
> ________________________________ From: Nofated <notifications@github.com> Sent: Wednesday, February 24, 2021 4:31:17 PM To: Gamuxorg/bbs <bbs@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: [Gamuxorg/bbs] 最新的Ubuntu无法打开城市天际线和造(tan)桥(wu)模拟器 (#141) 游戏名称： 操作系统：Ubuntu 20.10 x64 bug描述：打不开citiesskylines_1.7.0-f5_amd64和PolyBridge_amd64 终端运行输出 ： Set current directory to /tmp/.mount_VfD2Xh/usr/bin Found path: /tmp/.mount_VfD2Xh/usr/bin/Cities.x64 Mono path[0] = '/tmp/.mount_VfD2Xh/usr/bin/Cities_Data/Managed' Mono path[1] = '/tmp/.mount_VfD2Xh/usr/bin/Cities_Data/Mono' Mono config path = '/tmp/.mount_VfD2Xh/usr/bin/Cities_Data/Mono/etc' displaymanager : xrandr version warning. 1.6 client has 2 screens displaymanager screen (0)(LVDS-1): 1280 x 800 Using libudev for joystick management Importing game controller configs Set current directory to /tmp/.mount_PolyBrgRHhtX/usr/bin Found path: /tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge.x86_64 Mono path[0] = '/tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge_Data/Managed' Mono path[1] = '/tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge_Data/Mono' Mono config path = '/tmp/.mount_PolyBrgRHhtX/usr/bin/polybridge_Data/Mono/etc' displaymanager : xrandr version warning. 1.6 client has 2 screens displaymanager screen (0)(LVDS-1): 1280 x 800 Using libudev for joystick management Importing game controller configs 自行尝试解决bug的方式和结果：已经尝试通过公告<#97>中所给出的方案全部进行了安装 ― You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub<#141>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFRK4UOQUCAL4RDMBRRLNPDTAS2NLANCNFSM4YEB5J6A>.

有可能是图形库的问题

**Comment by n-nino** [comment]
用LD_DEBUG=lib调试一下看看

**Comment by n-nino** [comment]
我这里也有同样的问题，进入游戏之后载入，载入到一半退出。

**Comment by shouhuanxiaoji** [comment]
可能是打包问题，导致有一些目录权限变为只读，会尝试重新打包。
